libs/mapjet-core/src/lib/mapjet-core.spec.ts

```diff
@@ -262,6 +301,7 @@ describe('MapJet', () => {
           new MapJet({
             ...(val !== 'noKey' && { resizeObserver: val as any }),
             container: 'test',
+            style: '',
```

Comment:
Może powinniśmy to pole ustawić jako opcjonalne z domyślną wartością pustego stringa? :) 
Wtedy przy każdej instalacji oraz inicjalizacji pluginu nie będzie trzeba w app podawać tych styli? :)

Reply:
Podoba mi się ten pomysł tylko, że MapLibre wymaga podania styli w przeciwnym wypadku pojawia się trochę warningów w konsolce. Ostatnio o tym myślałem, i nie możemy inicjalizować w ten sposób naszej mapki jak mamy zrobione to aktualnie w aplikacji. Aktualnie trochę oszukujemy z tym pustym stringiem(choć w unit testach jest to jak najbardziej ok), i dodatkowo nie czekamy na ready state co do końca mi się nie podoba ;)

Reply:
Chociaż z drugiej strony skoro plugin i tak ustawia style to ten parametr prawie zawsze będzie ustawiany na pusty string podczas tworzenia instancji MapJet :smile: